observability/src/main/java/io/grpc/observability/interceptors/InternalLoggingChannelInterceptor.java

```diff
@@ -100,12 +88,14 @@ private InternalLoggingChannelInterceptor(LogHelper helper) {
     final Deadline deadline = LogHelper.min(callOptions.getDeadline(),
         Context.current().getDeadline());
 
-    // TODO (dnvindhya): implement isMethodToBeLogged() to check for methods to be logged
-    // according to config. Until then always return true.
-    if (!helper.isMethodToBeLogged(method.getFullMethodName())) {
+    if (!filterHelper.isMethodToBeLogged(method.getFullMethodName()).log) {
```

Comment:
How about passing `method` itself (type `MethodDescriptor`) to `filterHelper.isMethodToBeLogged` ?